source/user/pbc.rst

```diff
@@ -1,13 +1,36 @@
+.. _user_pbc:
 
 ****************************
 Periodic boundary conditions
 ****************************
 
+PySCF supports electronic structure calculations of
+extended systems with periodic boundary conditions (PBCs).  
+PBC-specific functionality must be imported from :mod:`pyscf.pbc`,
+which has a directory structure that mirrors that of the molecular :mod:`pyscf`
+module, e.g. ``from pyscf.pbc import gto, scf``.
+Details of how to specify the system details, including the unit cell,
+basis sets, and pseudopotentials, are provided in
+:ref:`user_pbc_gto`.
 
+All electronic structure methods can be applied with periodic
+boundary conditions at the Gamma point of the Brillouin zone, :math:`k=(0,0,0)`,
+as described in :ref:`mix_mol`.  Converging to the thermodynamic limit
```

Comment:
Should we say that we can use molecular methods at any point of the Brillouin zone? Although only k=(0,0,0) corresponds to PBC. If we have a twist averaging example in the examples directory, could link to that.